yarpl/test/FlowableTest.cpp

```diff
@@ -180,6 +180,18 @@ TEST(FlowableTest, SimpleTake) {
   EXPECT_EQ(std::size_t{0}, Refcounted::objects());
 }
 
+TEST(FlowableTest, SimpleSkip) {
+  ASSERT_EQ(std::size_t{0}, Refcounted::objects());
+  EXPECT_EQ(run(Flowables::range(0, 10)->skip(8)), std::vector<int64_t>({8, 9}));
```

Comment:
Can you also add a test for something like `range(0, 10)->skip(5)` but where the CollectingSubscriber only does `request(2)` and then `cancel()` when it gets the 2.  I believe `run()` does `request(100)` by default, which kind of makes the whole thing behave like an Observable for tests with < 100 items.